repository-info-extension/vss-extension.json

```diff
@@ -0,0 +1,67 @@
+{
+    "manifestVersion": 1,
+    "id": "repository-info-extension",
+    "version": "1.0.0",
+    "name": "Repository Information Sample",
+    "description": "Provides an example for accessing the currently selected repository from an extension.",
+    "publisher": "ms-samples",
+    "public": true,
+    "categories": [
+        "Developer samples"
+    ],
+    "links": {
+        "learn": {
+            "uri": "https://github.com/Microsoft/vso-extension-samples"
+        }
+    },
+    "icons": {
+        "default": "images/fabrikam-logo.png"
+    },
+    "targets": [
+        {
+            "id": "Microsoft.VisualStudio.Services"
+        }
+    ],
+    "content": {
+        "details": {
+            "path": "details.md"
+        }
+    }, 
+    "branding": {
+        "color": "rgb(190, 39, 3)",
+        "theme": "dark"
+    },       
+    "files": [
+        {
+            "path": "scripts",
+            "addressable": true
+        },
+        {
+            "path": "images",
+            "addressable": true
+        },
+        {
+            "path": "context.html",
+            "addressable": true
+        },                  
+        {
+            "path": "node_modules/vss-web-extension-sdk/lib",
+            "addressable": true
+        }    
+    ],
+    "contributions": [
+        {
+            "id": "code-hub",
```

Comment:
this ID is the same as the sample hub. we should probably differentiate it and also change the description to be more descriptive.